articles/openshift/oshift-faq.md

```diff
@@ -75,9 +75,9 @@ We are now required to self-assert our services, with customers taking responsib
 
 We provide confidence that our OpenShift service still meets the highest level of information assurance, which is why we continue to have our platform independently tested and validated, and have the findings made available to customers and partners. This enables SIROs to make an informed decision about any service they choose to consume.
 
-### Can I use UKCloud for OpenShift over closed networks such as PSN and N3/HSCN?
+### Can I use UKCloud for OpenShift over closed networks such as PSN and HSCN?
 
-We plan for this service to be accredited for use over PSN, however this may not be available at launch. Connectivity to the N3/HSCN network is supported.
+We plan for this service to be accredited for use over PSN, however this may not be available at launch. Connectivity to the HSCN network is supported.
```

Comment:
Already launched - need to change the tense of this sentence

Reply:
Fixed to cover current network connectivity